.cursor/commands/testcase.md

```
@@ -314,15 +314,15 @@
 **반드시 프로젝트 루트 경로에 마크다운 파일을 생성하세요!**
 
 - **파일명 형식**: `testcase-[이슈번호]-[간단한설명].md`
-  - 예: `testcase-407-닉네임변경.md`
-  - 예: `testcase-123-채팅기능.md`
+    - 예: `testcase-407-닉네임변경.md`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)